xrm/index.d.ts

```diff
@@ -474,6 +549,179 @@ declare namespace Xrm
          * A reference to the collection of tabs on the form.
          */
         tabs: Collection.ItemCollection<Page.Tab>;
+        
+        /**
+          * A collection of all the quick view controls on a form using the new form rendering engine (also called "turbo forms").
+          */
+        quickForms: Collection.ItemCollection<Page.ui.QuickForm>;
+    }
+
+    /**
+     * Interface for methods to create and manage records in the mobile clients (for phones tablets).
+     */
+    export interface Mobile
+    {
+        /**
+          * Provides methods to create and manage records in the mobile clients while working in the offline mode.
+          *
+          * @see {@link https://msdn.microsoft.com/en-us/library/mt787123.aspx} for details.
+          */
+        offline: MobileOffline;
+    }
+
+    /**
+     * Interface for the Mobile.offline methods to create and manage records in the mobile clients while working in the offline mode.
+     *
+     * @see {@link https://msdn.microsoft.com/en-us/library/mt787123.aspx} for details.
+     */
+    export interface MobileOffline
+    {
+        /**
+          * Returns whether an entity is offline enabled.
+          *
+          * @param    {string}    entityType    The logical name of the entity.
+          *
+          * @return   {boolean}   True if the entity is offline enabled; otherwise False.
+          */
+        isOfflineEnabled( entityType: string ): boolean;
+
+        /**
+          * Creates an entity record in mobile clients while working in the offline mode.
+          *
+          * @param    {string}    entityType    The logical name of the entity.
+          * @param    {Object}    data          A dictionary object containing key : value pairs for the record to create.
+          *
+          * @return   {Async.XrmPromise}        Returns an asynchronous promise.
+          *
+          * @remarks  You cannot create intersect and activity party entities.
+          *           Only the following attribute types are supported in offline mode: BigInt
```

Comment:
Can you start the list on a separate line so it doesn't need to be indented so much?